src/components/renderer/form-button.vue

```diff
@@ -9,40 +9,25 @@
 <script>
 import Mustache from 'mustache';
 import { getValidPath } from '@/mixins';
-import { isProxy } from 'is-proxy';
```

Comment:
@caleeli @josechirivella hey guys! Please consider removing this `"is-proxy": "^1.0.6",` dependency, since it was only used in this file. 👍

Reply:
@eiresendez thanks, dependency was removed.